backend/plugin/db/pg/dump.go

```diff
@@ -91,23 +93,60 @@ func (driver *Driver) dumpOneDatabaseWithPgDump(ctx context.Context, database st
 	args = append(args, "--no-owner")
 	// Avoid pg_dump v15 generate REVOKE/GRANT statement.
 	args = append(args, "--no-privileges")
-	args = append(args, database)
+	dbConnPairs = append(dbConnPairs, fmt.Sprintf("dbname=%s", database))
 
-	pgDumpPath := filepath.Join(driver.dbBinDir, "pg_dump")
-	cmd := exec.CommandContext(ctx, pgDumpPath, args...)
-	// Unlike MySQL, PostgreSQL does not support specifying commands in commands, we can do this by means of environment variables.
 	if driver.config.Password != "" {
-		cmd.Env = append(cmd.Env, fmt.Sprintf("PGPASSWORD=%s", driver.config.Password))
+		dbConnPairs = append(dbConnPairs, fmt.Sprintf("password=%s", driver.config.Password))
 	}
 	if driver.config.TLSConfig.SslCert != "" {
-		cmd.Env = append(cmd.Env, fmt.Sprintf("PGSSLCERT=%s", driver.config.TLSConfig.SslCert))
+		sslCertFile, err := os.CreateTemp(os.TempDir(), "pgsslcert")
```

Comment:
Should we consider the permission of file likes we connect to AWS ec2?

Reply:
I make file permission 0400. It should solve this permission problem.